applications/RomApplication/rom_application_variables.h

```diff
@@ -0,0 +1,32 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    @{KRATOS_APP_AUTHOR}
+//
+
+#if !defined(KRATOS_ROM_APPLICATION_VARIABLES_H_INCLUDED )
+#define  KRATOS_ROM_APPLICATION_VARIABLES_H_INCLUDED
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/define.h"
+#include "includes/variables.h"
+#include "includes/kratos_application.h"
+
+namespace Kratos
+{
+KRATOS_DEFINE_VARIABLE( int, AUX_ID )
```

Comment:
```suggestion
KRATOS_DEFINE_VARIABLE( std::size_t, AUX_ID )
```

Reply:
This change was not performed, as doing so resulted in importing errors. 
For the time being, the variable is kept as int.

Reply:
you also need to change it in the other places where you defined the variable